framework/aster-frame/src/task/processor.rs

```diff
@@ -62,8 +62,8 @@ pub fn schedule() {
 }
 
 pub fn preempt() {
-    // disable interrupts to avoid nested preemption.
-    let disable_irq = disable_local();
+    // TODO: Refactor `preempt` and `schedule`
+    // after the Atomic mode and `might_break` is enabled.
```

Comment:
We should make comments self-contained. It is not clear what is "Atomic mode" and "might_break" just by reading this comment.

My suggestion, but for references only:
```suggestion
    // TODO: Once kernel preemption is supported, we need to find out a proper way to avoid nested
    // preemption in `schedule()` and `preempt()`. We cannot simply disable preemption here. If we
    // do, the preemption counter will break after a task switch.
```

Reply:
@lrh2000  Good suggestion. But let's do the improvement in the atomic mode PR. I am merging this PR.